presto-main/src/main/java/io/prestosql/server/DynamicFilterService.java

```diff
@@ -67,6 +67,7 @@
 import static com.google.common.collect.ImmutableSet.toImmutableSet;
```

Comment:
you could make separate PR for this fix

Reply:
Raised https://github.com/prestosql/presto/pull/5099